woocommerce/woocommerce
# Allow enforce password strength meter on checkout

### All Submissions:






### Changes proposed in this Pull Request:


This PR introduce a filter to allow enforce password strength meter on checkout.
We disabled this on checkout to stop people from placing orders, but we still could allow shop managers to do it if they like.

Closes https://github.com/woocommerce/woocommerce/issues/23807.

### How to test the changes in this Pull Request:

1. Use `add_filter( 'woocommerce_enforce_password_strength_meter_on_checkout', '__return_true` );
2. Go the checkout as guest and try create weak password.
3. See that the "Place order" button is disabled.

### Other information:




### Changelog entry

> Dev - Introduced `woocommerce_enforce_password_strength_meter_on_checkout` filter to allow enforce password strength meter on checkout.
